### PR TITLE
Update README to redirect to DANDI CLI

### PR DESCRIPTION
**Pull request by aaronkanzer**
@kabilar -- let me know if you are fine with the wording of this -- ready to deprecate/archive once you are thumbs-up here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # LINC Brain Client
 
+## The LINC Brain Client has been deprecated as of November 2024. Users should instead refer to the [DANDI CLI PyPI package](https://pypi.org/project/dandi/) and [DANDI CLI repository](https://github.com/dandi/dandi-cli).
```
Comment (kabilar) [inline README.md:3]:
```suggestion
## The LINC Brain Client has been deprecated as of November 2024. Users should instead use the DANDI Client (`dandi`) with the `-i linc` argument.  See the DANDI Client [PyPI package](https://pypi.org/project/dandi/) and [GitHub repository](https://github.com/dandi/dandi-cli).  For further details, please refer to the [LINC Docs](https://docs.lincbrain.org/upload/).
```

Comment (aaronkanzer) [reply README.md:3]: Did you want just https://docs.lincbrain.org/ instead of https://docs.lincbrain.org/upload?

Comment (kabilar) [reply README.md:3]: Thanks.  I updated my suggestion.  I would suggest the `Upload` page.

**Comment by aaronkanzer** [comment]
@kabilar -- just confirming that https://pypi.org/project/lincbrain-cli/ displays the correct text

**Comment by kabilar** [comment]
> @kabilar -- just confirming that https://pypi.org/project/lincbrain-cli/ displays the correct text

That looks great.  Thanks @aaronkanzer.

**Pull request merged**